micropy/project/modules/packages.py

```diff
@@ -77,38 +78,17 @@ def context(self):
             'paths': _paths
         }
 
-    def _fetch_package(self, url):
-        """Fetch and stub package at url.
-
-        Args:
-            url (str): URL to fetch
-
-        Returns:
-            Path: path to package
-
-        """
-        with tempfile.TemporaryDirectory() as tmp_dir:
-            tmp_path = Path(tmp_dir)
-            file_name = utils.get_url_filename(url)
-            self.log.debug(f"Fetching package: {file_name}")
-            _file_name = "".join(self.log.iter_formatted(f"$B[{file_name}]"))
-            content = utils.stream_download(
-                url, desc=f"{self.log.get_service()} {_file_name}")
-            pkg_path = utils.extract_tarbytes(content, tmp_path)
-            ignore = ['setup.py', '__version__', 'test_']
-            pkg_init = next(pkg_path.rglob("__init__.py"), None)
-            py_files = [f for f in pkg_path.rglob(
-                "*.py") if not any(i in f.name for i in ignore)]
-            stubs = [utils.generate_stub(f) for f in py_files]
-            if pkg_init:
-                data_path = self.pkg_path / pkg_init.parent.name
-                shutil.rmtree(data_path, ignore_errors=True)
-                shutil.copytree(pkg_init.parent, data_path)
-                return data_path
-            # Iterates over flattened list of stubs tuple
-            file_paths = [(f, (self.pkg_path / f.name)) for f in list(sum(stubs, ()))]
-            for paths in file_paths:
-                shutil.move(*paths)  # overwrite if existing
+    def install_package(self, source: Union[LocalDependencySource, PackageDependencySource]) -> Any:
+        with source as files:
+            if isinstance(files, list):
+                self.log.debug(f"installing {source} as module(s)")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Instance of 'PackagesModule' has no 'log' member](https://app.codacy.com/app/BradenM/micropy-cli/pullRequest?prid=4593453)